QuickLayout.Touch/Views/UpdateConstraintsView.cs

```diff
@@ -17,46 +17,50 @@ public override void ViewDidLoad()
 
 			View.BackgroundColor = UIColor.White;
 
-			if (RespondsToSelector(new Selector("edgesForExtendedLayout")))
-				EdgesForExtendedLayout = UIRectEdge.None;
+		    if (RespondsToSelector(new Selector("edgesForExtendedLayout")))
+		    {
+		        EdgesForExtendedLayout = UIRectEdge.None;
+		    }
 
-			var label = new UILabel 
+		    var label = new UILabel 
 			{ 
-				Text = "Update this label's height constraint height constant and active settings",
+				Text = "This label has a Height constraint applied to it. \r\nThe toggle switch is bound to the Contraint's Active boolean and the slider to the Constraint's Constant value.",
```

Comment:
Not a big deal but technically you only need `\n` on iOS. A good rule of thumb is just to use `Environment.NewLine` and then you never need to remember, especially now that C# supports string interpolation 😄